frontend/src/app/components/shared/upload/song-image/song-image.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="uploadblock">
+    <div class="dropZone" appDragDrop [type]="fileType" [addedFiles]="files" (files)="filesDropped($event)">
+        <h3 class="drag-text"><i class="cloud upload icon"></i></h3>
+        <h3 class="drag-text body-regular">Drag and drop file here</h3>
+        <h4 class="drag-text body-regular">or you can</h4>
+        <div class="huge ui labeled icon button btn-full upload-button">
+            <i class="upload icon" aria-hidden="true"></i>
+            Upload
+            <input type="file" multiple class="filepicker" [attr.accept]="fileType===1 ? 'image/*' : 'audio/*' " (change)="filesAddedByButton($event)">
```

Comment:
Add spaces in fileType===1
Refactor to avoid numbers in file types. This is not obvious for a stranger